system/jlib/jutil.cpp

```diff
@@ -71,6 +71,29 @@ static CriticalSection * protectedGeneratorCs;
 mach_timebase_info_data_t timebase_info  = { 1,1 };
 #endif
 
+bool getEnvVar(const char * varName, StringBuffer & varValue)
+{
+    if (isEmptyString(varName))
+        return false;
+
+    try
+    {
+        varValue.set(std::getenv(varName));
+        if (varValue.isEmpty())
+        {
+            DBGLOG("Could not find environment variable '%s'!", varName);
```

Comment:
minor: I still think this is too verbose. This is a general purpose function that returns false if it can't get the env. variable requested.